components/push/src/push_msg_types.proto

```diff
@@ -8,6 +8,8 @@ option java_outer_classname = "MsgTypes";
 message DispatchInfo {
     required string uaid = 1;
     required string scope = 2;
+    required string endpoint = 3;
+    required string appServerKey = 4;
```

Comment:
Shouldn't the Kotlin data class also expose these fields since it isn't an auto-generated Kotlin class?

Everything else looks straight-forward to me. ✅

Reply:
d'oh. you're right. Adding those.